server_admin/topics/threat/host.adoc

```diff
@@ -87,6 +87,11 @@ By default the `httpPort` and `httpsPort` are received from the request. As long
 it should not be necessary to change this. It is possible to configure fixed ports if necessary by setting the `httpPort` and
 `httpsPort` properties on the fixed provider.
 
+In most cases the scheme should be set correctly. This may not be true if the reverse proxy is unable to set the `X-Forwarded-For` header
+correctly, or if there is an internal application using non-https to invoke {project_name}. The latter is not recommended
+and you should strive to use `https` always. In this case you can set a fixed scheme as well by setting the `scheme`
```

Comment:
I would phrase this "...you should strive to always use `https`." but this is not significant enough to prevent approval.